Packages/doc/installation.rst

```diff
@@ -195,3 +199,28 @@ hardware related XOPs present.
    -  In ``Igor Help Files`` a shortcut pointing to HelpFiles-IP9
 
 -  Start Igor Pro
+
+MacOSX (without hardware)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We do support analysis on MacOSX only. We neither support data acquisition nor
```

Comment:
Analysis support only. Data acquisition and NWBv2 export are not supported on MacOSx.

Reply:
Fixed.